weaver/common/protos-rs/pkg/src/generated/common.ack.rs

```diff
@@ -1,7 +1,3 @@
-// Copyright IBM Corp. All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-
```

Comment:
Please add these copyright statements back, in this file and others.

Reply:
@VRamakrishna added back. I didn't mean to remove the copyright statement. It is a bit strange because it seems that these files are automatically generated. We might need an automated way to add this statement every time they are generated.

Reply:
@outsidethecode Good point. We'll add logic in the Makefile to add the copyright statements for auto-generated files.